pkg/manager/options.go

```diff
@@ -139,4 +142,13 @@ func (o *Options) defaults() {
 	if o.Password == "" {
 		o.Password = os.Getenv("VSPHERE_PASSWORD")
 	}
+	if ns, ok := os.LookupEnv("POD_NAMESPACE"); ok {
+		o.PodNamespace = ns
+	} else if data, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace"); err == nil {
+		if ns := strings.TrimSpace(string(data)); len(ns) > 0 {
+			o.PodNamespace = ns
+		}
+	} else {
+		o.PodNamespace = DefaultPodNamespace
```

Comment:
same, used ?

Reply:
same as above.

follow up issue to track, wdyt?

Reply:
Sgtm